integration-tests/src/test/java/org/apache/druid/tests/coordinator/duty/ITAutoCompactionTest.java

```diff
@@ -641,7 +643,45 @@ public void testAutoCompactionDutyWithRollup() throws Exception
     }
   }
 
+  @Test
+  public void testAutoCompactionDutyWithQueryGranularity() throws Exception
+  {
+    Map<String, Object> specs = ImmutableMap.of("%%GRANULARITYSPEC%%", new UniformGranularitySpec(Granularities.DAY, Granularities.NONE, true, ImmutableList.of(new Interval("2013-08-31/2013-09-02"))));
```

Comment:
Looks like the `Interval#<init>(java.lang.Object)` usage here and earlier are tripping the forbidden API checks.

Reply:
Fixed!